spec/unit/compressors/dmg_spec.rb

```diff
@@ -3,6 +3,7 @@
 module Omnibus
   describe Compressor::DMG do
     let(:project) do
+      Config.append_timestamp(false)
```

Comment:
It might be easier to set `Config.append_timestamp(false)` option globally here:
https://github.com/chef/omnibus/blob/398b1d8b59267ceec2506b53e7e66eb5aacfe9cf/spec/spec_helper.rb#L58

This would save having to change all these specs.